controllers/apply/form-router/steps.js

```diff
@@ -69,7 +70,11 @@ module.exports = function(formId, formBuilder) {
          * redirect to the first step in the section.
          */
         if (!stepNumber) {
-            return res.redirect(`${res.locals.formBaseUrl}/${section.slug}/1`);
+            return res.redirect(
+                `${res.locals.formBaseUrl}/${section.slug}/1${
```

Comment:
Not sure about this line, do we need this logic?

Reply:
Perhaps not anymore, it was only relevant when you could click the section titles from the summary screen (whereas now it's just the "Change" links). Will remove.